buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -281,13 +297,33 @@ def sanitize_node(nodeName) {
         println(e.getMessage())
     }
 
+    //reconnect slave
     timeout(time: RECONNECT_TIMEOUT.toInteger(), unit: 'MINUTES') {
-        println("\t ${nodeName}: Reconnecting...")
+        println("\t ${nodeName}: Disconnecting...")
+        waitUntil {
+            // wait for Jenkins to disconnect the slave after kill -9 or reboot
+            return (workingComputer.getNode().getRootPath() == null)
```

Comment:
Technically if the node is back online before we get to this point we would get stuck in this loop. I very highly doubt that would happen though.

Reply:
thus the timeout